examples/next-openai-pages/app/api/stream-chat/route.ts

```diff
@@ -1,13 +1,13 @@
-import { CoreMessage, streamText } from 'ai';
+import { convertToCoreMessages, streamText } from 'ai';
 import { openai } from '@ai-sdk/openai';
 
 export async function POST(req: Request) {
-  const { messages }: { messages: CoreMessage[] } = await req.json();
+  const { messages } = await req.json();
 
   const result = await streamText({
-    model: openai('gpt-4'),
+    model: openai('gpt-4o'),
     system: 'You are a helpful assistant.',
-    messages,
+    messages: convertToCoreMessages(messages),
   });
 
   return result.toAIStreamResponse();
```

Comment:
instead of adding to `next-openai-pages`, what do you think about adding to `next-openai` as `use-chat-attachments`?

Reply:
moved the examples to `next-openai`!